src/tests/e2etest/CommandRunnerTests.cs

```diff
@@ -1,19 +1,279 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using Microsoft.Azure.Devices;
+using Microsoft.Azure.Devices.Shared;
 using NUnit.Framework;
 using System;
 using System.Text.Json;
 using System.Threading.Tasks;
 
 namespace E2eTesting
 {
-    public class CommandRunnerTests : E2eTest
+
+    [TestFixture]
+    public abstract class E2ETest
     {
-        private const string componentName = "CommandRunner";
-        private const string commandArguments = "CommandArguments";
-        private const string commandStatus = "CommandStatus";
+        private RegistryManager _registryManager;
+
+        private readonly string _iotHubConnectionString = Environment.GetEnvironmentVariable("E2E_OSCONFIG_IOTHUB_CONNSTR")?.Trim('"');
+        private readonly string _moduleId = "osconfig";
+        private readonly string _deviceId = Environment.GetEnvironmentVariable("E2E_OSCONFIG_DEVICE_ID");
+
+        private readonly string _sasToken = Environment.GetEnvironmentVariable("E2E_OSCONFIG_SAS_TOKEN");
+        private readonly string _uploadUrl = Environment.GetEnvironmentVariable("E2E_OSCONFIG_UPLOAD_URL")?.Trim('\"');
+        private readonly string _resourceGroupName = Environment.GetEnvironmentVariable("E2E_OSCONFIG_RESOURCE_GROUP_NAME")?.Trim('\"');
+
+        public partial class GenericResponse<T>
```

Comment:
Why is GenericResponse defined as a partial class?

Reply:
Good catch, this does not need to be marked as `partial`(was leftover from when I was testing something).